src/Misc/externals.sh

```diff
@@ -8,6 +8,7 @@ CONTAINER_URL=https://vstsagenttools.blob.core.windows.net/tools
 NODE_URL=https://nodejs.org/dist
 NODE_VERSION="6.17.1"
 NODE10_VERSION="10.21.0"
+NODE14_VERSION="14.10.0"
```

Comment:
This is the latest as of this PR. We will probably change this once 14 hits LTS in October.

Reply:
We periodically bump this (within a major version) so that seems like a good plan.